CHANGES.rst

```diff
@@ -141,6 +142,8 @@ Bug Fixes
 
 - ``astropy.io.fits``
 
+  - Fixed ``fits.Header._keyword_from_index()`` to give correct keyword and repeat number [#5827]
```

Comment:
1. Change log for bug fix should fall under 1.3.1. 
2. Include this PR number (not just the original issue). For example, `[#5827, #5852]`. Also does not hurt to add a period at the end of "number".